manager/assets/modext/workspace/lexicon/lexicon.grid.js

```diff
@@ -56,7 +57,7 @@ MODx.grid.Lexicon = function(config) {
             xtype: 'modx-combo-namespace'
             ,id: 'modx-lexicon-filter-namespace'
             ,itemId: 'namespace'
-            ,preselectValue: MODx.request['ns'] ? MODx.request['ns'] : ''
+            ,preselectValue: MODx.request.ns || 'core'
```

Comment:
What if user doesn't have access to the `core` namespace?

Reply:
An interesting question, can this also be in the "System settings"? Because there is now `core` specified (and was originally) by default - https://github.com/modxcms/revolution/blob/3.x/manager/assets/modext/widgets/core/modx.grid.settings.js#L35.

Reply:
Yes, seems like it'll have the same issue - you'll load with an empty grid

Reply:
And what is the solution? It is worth changing the line to: `preselectValue: MODx.request.ns || 'core' || ''` ?
Or can we leave it as it is? Not sure if there will be any error, or an empty mesh will just load.

Reply:
@Jako can you have any thoughts on this?

Reply:
The core restriction should better not be added here.

It is quite annoying implemented in the system settings grid (at least in 2.8.x): It still shows `core` there when the search is active but then this filter is not regarded there anymore. And even worse: It shows and filters the first available namespace if the core namespace is on page 2 in the namespace combo. This is normally not visible but on my dev machine I have a lot of namespaces.

Reply:
As I understand it, `preselectValue` sets only the text in the "namespace" drop-down list, the value is not involved in the request.
Moreover, even if i remove the `'core'` value in the `core/src/Revolution/Processors/Workspace/Lexicon/GetList.php` and in `baseParams` in `manager/assets/modext/workspace/lexicon/lexicon.grid.js`, the `core` lexicons are still displayed in the grid.
@theboxer @Jako Can you send edits to this PR? Because I do not understand what and how the grid should work in the end.

Reply:
Guys, I suggest to merge this PR, because. PR improves MODX - adds filters to the URL, and does not create new issues.
And for the question about the 'core' namespace, it's worth creating a separate issue on github, because 'core' is also found in System Settings, possibly in other sections as well.

Reply:
@theboxer you requested changes, are they still needed?